src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -472,6 +484,16 @@ void AutomatedRssDownloader::handleRuleCheckStateChange(QListWidgetItem *ruleIte
     m_ui->listRules->setCurrentItem(ruleItem);
 }
 
+void AutomatedRssDownloader::clearDownloadedEpisodesSelectedRule()
+{
+    QMessageBox::StandardButton reply = QMessageBox::question(this, tr("Clear downloaded episodes"), tr("Are you sure you want to clear the downloaded episodes for the selected rule?"), QMessageBox::Yes | QMessageBox::No);
```

Comment:
@evsh:  As in `the downloaded episode list` or `the list of downloaded episodes`?  

Both are technically correct in English, but the latter option is clearer and easier to translate into other languages.